Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2887,6 +2887,7 @@ extension BrowserViewController: HomeMenuControllerDelegate {
     }
     
     func menuDidBatchOpenURLs(_ menu: HomeMenuController, urls: [URL]) {
+        menu.dismiss(animated: true)
```

Comment:
Probably best to move this to the bottom of the function, so tab loads start first.